packages/vitest/src/defaults.ts

```diff
@@ -42,6 +42,7 @@ export const coverageConfigDefaults: ResolvedCoverageOptions = {
 export const browserConfigDefaults = {
   enabled: false,
   headless: isCI,
+  name: 'chrome',
```

Comment:
@sheremet-va Any way I can avoid this? If I don't put it the type complains (which is expected kinda)